Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.5.5-jessie
+
+# TODO: move in a base image
+RUN useradd -ms /bin/bash -G www-data elife && \
+    chown elife:elife /srv && \
+    mkdir /srv/bin && \
+    chown elife:elife /srv/bin && \
+    mkdir -p /var/www && \
+    chown www-data:www-data /var/www
+
+RUN pip install virtualenv
+RUN pip install -U setuptools
+# END base image
+
+USER elife
+ENV PROJECT_FOLDER=/srv/profiles
+RUN mkdir ${PROJECT_FOLDER}
+WORKDIR /srv/profiles
+COPY --chown=elife:elife install.sh requirements.txt /srv/profiles/
+RUN PROFILES_SKIP_DB=1 /bin/bash install.sh
```

Comment:
bare minimum installation of dependencies, this layer should be cached despite changes to the code in `profiles/`

Reply:
This is ignoring the DB for now?

Reply:
Yes, I am checking out how Init Containers work in Kubernetes (and whether they can be ported to docker-compose) to copy the pattern. We a standard way to make database migration happen on startup, rather than at image build time.

Reply:
(because the `postgres` image itself would have a volume containing the data, so every new deployment needs to re-run the setup)